Source/cmake/OptionsGTK.cmake

```diff
@@ -303,14 +302,12 @@ if (ENABLE_DOCUMENTATION AND NOT GIDocgen_FOUND)
     message(FATAL_ERROR "ENABLE_INTROSPECTION is needed for gi-docgen.")
 endif ()
 
-if (ENABLE_WEB_CRYPTO)
-    find_package(Libtasn1 REQUIRED)
-    if (NOT LIBTASN1_FOUND)
-        message(FATAL_ERROR "libtasn1 is required to enable Web Crypto API support.")
-    endif ()
-    if (LibGcrypt_VERSION VERSION_LESS 1.7.0)
-        message(FATAL_ERROR "libgcrypt 1.7.0 is required to enable Web Crypto API support.")
-    endif ()
+find_package(Libtasn1 REQUIRED)
+if (NOT LIBTASN1_FOUND)
+    message(FATAL_ERROR "libtasn1 is required to enable Web Crypto API support.")
+endif ()
+if (LibGcrypt_VERSION VERSION_LESS 1.7.0)
+    message(FATAL_ERROR "libgcrypt 1.7.0 is required to enable Web Crypto API support.")
 endif ()
```

Comment:
Since these deps are required now, please move the find_package() call for Libtasn1 to the top of the file in the alphabetized list of required packages, and just add the 1.7.0 version requirement to the libgcrypt find_package() call. We no longer need to have custom error messages and they don't make sense anymore since there's no way to disable the feature. Same comment applies to both OptionsGTK.cmake and OptionsWPE.cmake as well.